.pre-commit-config.yaml

```diff
@@ -0,0 +1,28 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.2.0
+    hooks:
+    -   id: trailing-whitespace
+    -   id: end-of-file-fixer
+    -   id: check-yaml
+    -   id: check-added-large-files
+
+-   repo: https://github.com/pycqa/isort
+    rev: 5.10.1
+    hooks:
+    -   id: isort
+
+-   repo: https://github.com/psf/black
+    rev: stable
+    hooks:
+    -   id: black
+
+-   repo: https://github.com/gvanderest/pylama-pre-commit
+    rev: 0.1.2
+    hooks:
+    -   id: pylama
+
+-   repo: https://github.com/pre-commit/mirrors-mypy
+    rev: ''
```

Comment:
Why is it blank? it makes sense to use same revisions as in `pyproject.toml` here